containers/FeedList/components/Tag.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import styled from "@emotion/styled";
+import colors from "@/styles/colors";
+
+interface Props {
+    name: string
+    isActive?: boolean
+    onClick?: () => void
+}
+
+const Tag = ({name, isActive, onClick}: Props) => {
+    return (<Container isActive={isActive ?? false} onClick={onClick}>
```

Comment:
```suggestion
    return (<Container isActive={isActive} onClick={onClick}>
```
isActive는 어차피 boolean이라서 ??없이 써도 됩니다.